tools/comm-test/src/lighthouse.rs

```diff
@@ -0,0 +1,214 @@
+use base64::prelude::*;
+use pgp::{composed::SignedSecretKey, types::SecretKeyTrait};
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use std::{
+    net::Ipv4Addr,
+    str::{from_utf8, FromStr},
+};
+use string_comm::crypto::{Crypto, SigningError};
+use thiserror::Error;
+
+#[allow(dead_code)]
+#[derive(Error, Debug)]
+pub enum LighthouseClientError {
+    #[error("unknown error")]
+    Unknown,
+    #[error("failed to make web request")]
+    RequestError(#[from] reqwest::Error),
+    #[error("failed to export pubkey")]
+    PubkeyError(#[from] pgp::errors::Error),
```

Comment:
```suggestion
    PubKeyError(#[from] pgp::errors::Error),
```

Reply:
Less concerned about these two since they're internal to comm-test, but still good practice.